### DISCUSSION TITLE
qq新设备认证提示账号被冻结

### DISCUSSION THREAD

**Discussion by moyulanxin** [Arondight/Adachi-BOT#386] [Q&A]
[安全提醒]该帐号因涉嫌发布/传播诈骗信息或涉嫌诈骗行为被暂时冻结，请点
击下方按钮了解详情及处理。(错误码：40)试过好几个qq号，都是提示这个。

**Comment by mark9804** [comment]
……你确定这几个账户真的没问题？正常被封一般是这个画风的，我们项目和诈骗行为可不搭边啊
![image](https://user-images.githubusercontent.com/9006264/141276434-22a56cbf-de98-4df5-953d-d92b7a439836.png)

**Comment by Arondight** [comment]
手机登一下号看看有没有问题，有问题的话是 QQ 本来就被封了，没问题去看一下配置文件是不是读到了你自己的，还是用的默认的 987654321
